config-templates/ibm/ibm-object-storage-plugin/2.2/metadata.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "2.2",
+  "revision": "1", 
+  "enabled": "true",
+  "status": "supported",
+  "description": "[Beta] - IBM Object Storage Plugin version 2.2.1",
+  "tags": [
+    {
+      "name": "provider",
+      "value": "ibm"
+    },
+    {
+      "name": "volume-type",
+      "value": "object"
+    },
+    {
+      "name": "provider-logo",
+      "value": "https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/ibm/ibm-object-storage-plugin/2.2/ibm-object-storage-plugin.svg"
```

Comment:
Use relative path similar to https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/azure/azurefile-csi-driver/1.22.0/metadata.json#L30